core/src/main/kotlin/no/strooped/view/screen/TaskScreen.kt

```diff
@@ -22,15 +22,20 @@ import no.strooped.view.screen.components.Label
 private val FONT_SIZE_LABEL_TEXT = TextureSizes.getScaledFontSize(3.0f)
 class TaskScreen(
     private val controller: StroopedController,
-    private val gameRoom: GameRoom
+    private val currentTask: Task
 ) : Screen {
     private var ui: Stage = Stage(ScreenViewport())
-    private val backgroundPosition = Vector2(0f, 0f)
     private val background: Image = Image(Texture("white.jpg"))
-    private var colorOptions: Array<ColorButton?> = arrayOfNulls(gameRoom.currentTask!!.possibleAnswers.size)
+    private var colorOptions: Array<ColorButton?> = arrayOfNulls(currentTask.buttons.size)
```

Comment:
I see we instead check how many buttons we use, which is probably a more flexible solution. But do we then need the constnt `numberOfColors` in TextureSizes or could we remove it?

Reply:
It has to stay in texturesizes since it changes the size of the colors (more colours, smaller size)

Reply:
But wouldn't that create some inconsistencies between our constant and the actual number of colors. Say our constant is 4, but we receive 5 or 6 items (although highly unlikely). Wouldn't that screw up the layout anyhow?

Reply:
yes, that's what I noted earlier, I'll fix it right now